get.sh

```diff
@@ -367,6 +367,12 @@ getBinaryOpenjdk()
 	for file_name in "${jdk_file_array[@]}"
 	do
 		if [[ ! "$file_name" =~ "sbom" ]]; then
+			if [[ $file_name == *xz ]]; then
```

Comment:
Can we make the check more specific? Instead of checking `xz`, check
`if [[ $file_name == *.tar.xz ]]; then`

Reply:
The primary reason for not doing that is that .txz is also used by some people (same as `.tgz` is often used as a shorted `.tar.gz`), so this would catch that situation too .